src/components/inputs/creatable-select-input/creatable-select-input.js

```diff
@@ -135,25 +181,37 @@ export class CreatableSelectInput extends React.Component {
                   }
                 : undefined
             }
+            onChange={(value, info) =>
+              // selectedOptions is either an array, or a single option
+              // depending on whether we're in multi-mode or not (isMulti)
```

Comment:
... and? what do we do with this information when it comes to `onChange`?
Or was this comment meant to clarify that `value` is either __one of__ options or __the__ option provided depending on `isMulti`?

Reply:
The `value` argument used to be called `selectedOptions`. This was confusing as `selectedOption` wasn't always an array (as the name misleadingly indicated). That's why the comment was added.

The comment is only there for the next person looking at this code to understand that `value` can be an array or a single value depending on `isMulti`. We are just forwarding the value but I felt like it is good to know anyways.